lib/ts/BaseLogFile.h

```diff
@@ -22,11 +22,11 @@
 
  */
 
-#ifndef BASE_LOG_FILE_H
-#define BASE_LOG_FILE_H
+#pragma once
```

Comment:
Why `#pragma once`? If we want to make changes like this, I think it is best to make it once everywhere, rather than have a drawn-out period of inconsistency.